packages/yourloops/pages/product-labelling/product-labelling-page.tsx

```diff
@@ -29,37 +29,44 @@ import React, { type FC, useState } from 'react'
 import parse from 'html-react-parser'
 import { getCurrentLang } from '../../lib/language'
 import i18n from 'i18next'
-import rawHtmlEN from './raw-html/EN'
-import rawHtmlFR from './raw-html/FR'
-import rawHtmlES from './raw-html/ES'
-import rawHtmlDE from './raw-html/DE'
-import rawHtmlNL from './raw-html/NL'
-import rawHtmlIT from './raw-html/IT'
 import Box from '@mui/material/Box'
 import { LanguageCodes } from '../../lib/auth/models/enums/language-codes.enum'
 import { setPageTitle } from '../../lib/utils'
 import { useTranslation } from 'react-i18next'
+import config from '../../lib/config/config'
 
+function getFileName(): string {
+  switch (getCurrentLang()) {
+    case LanguageCodes.Fr :
+      return config.YLPZ_RA_LAD_FR
+    case LanguageCodes.En :
+      return config.YLPZ_RA_LAD_EN
+    case LanguageCodes.De :
+      return config.YLPZ_RA_LAD_DE
+    case LanguageCodes.Nl :
+      return config.YLPZ_RA_LAD_NL
+    case LanguageCodes.It :
+      return config.YLPZ_RA_LAD_IT
+    case LanguageCodes.Es :
+      return config.YLPZ_RA_LAD_ES
+    default:
+      return config.YLPZ_RA_LAD_EN
+  }
+}
 export const ProductLabellingPage: FC = () => {
   const { t } = useTranslation()
   const getHtml = (): string => {
-    switch (getCurrentLang()) {
-      case LanguageCodes.De:
-        return rawHtmlDE
-      case LanguageCodes.Es:
-        return rawHtmlES
-      case LanguageCodes.Fr:
-        return rawHtmlFR
-      case LanguageCodes.Nl:
-        return rawHtmlNL
-      case LanguageCodes.It:
-        return rawHtmlIT
-      case LanguageCodes.En:
-      default:
-        return rawHtmlEN
-    }
+
+    return `
+        <object
+            role="document"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏We don't need to have a separate method to handle the `object` tag, we can put it directly in the main `return`.

To do that, we need to:
- Remove the state based on the `html` and create a state based on the `filename` (it's the only thing that depends on the language)
```ts
const [filePath, setFilePath] = useState(getFilePath())
```
- Create a method to get the whole file path
```ts
const getFilePath = (): string => {
  return `${config.ASSETS_URL}${getFileName()}.pdf`
}
```
- Update the listener on the language to update the file path (and not the html) when the language changes
```ts
i18n.on('languageChanged', () => {
  setFilePath(getFilePath())
})
```
- And finally, remove the method `getHtml()` and put the `object` tag in the main `return()` of the page, and make it depend on the `filePath` state
```ts
<Box marginBottom={2} sx={{ width: '100%', height: '100%' }}>
  <object
    role="document"
    type="application/pdf"
    data={filePath}
    width="100%" height="100%"
  />
</Box>
```